runtime/doc/builtin.txt

```diff
@@ -8975,8 +8978,22 @@ str2nr({string} [, {base} [, {quoted}]])			*str2nr()*
 
 		Can also be used as a |method|: >
 			GetText()->str2nr()
+<
+strbyteindex({string} [, {index} [, {use_utf16}])	*strbyteindex()*
+		Convert a UTF-32 or UTF-16 {index} to a byte index. If
```

Comment:
Sometimes the doc in the PR uses "UTF-16" and sometimes "utf-16".
Let's be consistent (the capitalized one is better IMO).